lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -138,11 +138,13 @@ class LighthouseReportViewer {
   _replaceReportHtml(json) {
     this._validateReportJson(json);
 
-    if (!json.lighthouseVersion.startsWith('3')) {
+    if (json.lighthouseVersion.startsWith('2')) {
```

Comment:
Not sure about this section, but do we need to worry that the 3x viewer now won't render a 4x LHR (b/c the crc children removal?)  Is that what line 146-147 are?

Reply:
when we deploy-viewer that'll update the _default_ viewer which should handle 3x and 4x fine (incl crc). That's my running expectation.

@brendankenny will have to be smart about choosing to use to the PWACategoryRenderer